src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss'
+
+export const CommentList = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments?.map(comment => (
```

Comment:
```suggestion
    <div className="CommentList">
      {comments.map(comment => (
```
Same here. Comments should be required in this component. 
There is no need to render CommentList if there are no comments